server/protocols/jmap-rfc-8621/pom.xml

```diff
@@ -78,6 +78,12 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>net.javacrumbs.json-unit</groupId>
+            <artifactId>json-unit-assertj</artifactId>
+<!--            <type>test-jar</type>-->
+<!--            <scope>test</scope>-->
```

Comment:
You need the test scope IMO

Reply:
I tested with `test scope` but it cannot used, idk why?